spec_linux.go

```diff
@@ -127,6 +127,13 @@ type CPU struct {
 	Mems string `json:"mems"`
 }
 
+type Pids struct {
+	// Maximum number of PIDs. A value <= 0 implies "no limit".
+	// While technically 0 means "no tasks allowed in the cgroup", this does
+	// not make sense in the context of containers.
```

Comment:
If the pid limit is not immutable, it might make sense to set to 0 for some use cases. Should we be restricting the input domain in the spec?